src/tanks/Config.java

```diff
@@ -9,24 +9,34 @@
     private static double percCros = 0.3;
     private static double percMuta = 0.3;
     private static double percNew = 0.2;
-    public static int numOfGenes = 5;
-    public static int numOfCommands = 5;
+    public static int numOfGenes = 9;
+    public static int numOfCommands = 9;
+    public static double percRun = 0.2;
+    public static double percOnHit = 0.4;
+    public static double percOnScanned = 0.4;
 
-    public void Config(int pop, int iter, double percBest, double percCros, double percMuta, double percNew, int numOfGenes, int numOfCommands){
+
+    public void Config(int pop, int iter, double percBest, double percCros, double percMuta, double percNew, int numOfGenes, int numOfCommands, double percRun, double percOnHit, double percOnScanned){
         this.popSize = pop;
         this.iter = iter;
         this.numOfGenes = numOfGenes;
 
-        if(percBest + percCros + percMuta + percNew == 1) {
+        if(percBest + percCros + percMuta + percNew + percRun + percOnHit + percOnScanned == 1) {
```

Comment:
Do tohoto Ifu by nemely patrit staticke promenne percRun, percOnHit, percOnScanned, protoze tyto hodnoty patri do jine 'skupiny' nez prvni ctyri. Dava smysl vytvorit zvlast stejne stavenou podminku If pro tyto hodnoty :).

Reply:
Ona cela skupina percBest atd by se měla vyhodit. Nyní už nedává smysl. Máme to rozděleno, že bereme 76 procent nejlepších a následně z nich děláme 76 kříženců a dodáváme dalších 24 náhodně generovaných